extensions/azdata/src/azdata.ts

```diff
@@ -62,7 +62,7 @@ export class AzdataTool implements azdataExt.IAzdataApi {
 
 	public arc = {
 		dc: {
-			create: (namespace: string, name: string, connectivityMode: string, resourceGroup: string, location: string, subscription: string, profileName?: string, storageClass?: string): Promise<azdataExt.AzdataOutput<void>> => {
+			create: (namespace: string, name: string, connectivityMode: string, resourceGroup: string, location: string, subscription: string, profileName?: string, storageClass?: string, additionalEnvVars?: { [key: string]: string }): Promise<azdataExt.AzdataOutput<void>> => {
```

Comment:
These should all use the AdditionalEnvVars type you added

Reply:
Done